### ISSUE TITLE
h265bitstream support

### ISSUE THREAD

**Issue by leslie-wang** [aizvorski/h264bitstream#21]
:-) this is a good job. hope it can support h.265 too.

**Comment by aizvorski** [comment]
@leslie-wang How much code would be shared between the two?  It may be a whole new library :)  although of course a lot of the structure/style can be copied over.

**Comment by leslie-wang** [comment]
yes, I like your perl script a lot. The whole structure is extremely useful, and should be common for all codecs, such as 264, 265, VC9, coming AC1. :-). I think it is better to share the same structure, and different codecs just have different .in file. Even further, ideally, :-), each codec probably just provide syntax syntax tabular form, and one script can auto-generate all codes as you have done.

**Comment by leslie-wang** [comment]
@aizvorski Apples starts to support HEVC in IOS from this year WWDC. It is time to work on this feature now. I'll do some work, and send a pull review

**Comment by paulboschert** [comment]
@leslie-wang I just started to do some research on this, may I work with you?

**Comment by leslie-wang** [comment]
@paulboschert Welcome collaboration definitely. Are you familiar with CABAC, which probably need most help there? :-)

**Comment by leslie-wang** [comment]
@aizvorski I have committed HEVC support in this repo https://github.com/leslie-wang/hevcbitstream. I realized most syntax (even some const variable, such as slice_type) has been changed. so it is a little bit difficult to merge into existing h264bitstream project. I'm wondering if we need a new repo to combine this 2 and also leave extensibility for future video codec. What do you think?
